tour-of-go/exports/data/data.go

```diff
@@ -0,0 +1,15 @@
+package data
+
+import "fmt"
+
+var Happy string = "my name is happy"
+
+func Class() {
+	class12 := class(45, 23)
+	fmt.Println("The number of class1 and class2 = ", class12)
+}
+
+func FirstName() {
```

Comment:
According to what I taught you yesterday, correct this.

Function name must be a verb.